cross/libtorrent-rakshasa/Makefile

```diff
@@ -1,18 +1,13 @@
 PKG_NAME = libtorrent
-PKG_VERS = 0.13.8
+PKG_VERS = 0.14.0
```

Comment:
A version 0.15.4 is available

Reply:
I note that in an earlier commit (https://github.com/SynoCommunity/spksrc/pull/6404/commits/0e69755b14e3570cbd194858216a999d1df8d166), `libtorrent` was explicitly downgraded from 0.15.1 to 0.14.0, so I am hesitant to use 0.15.x. @smaarn, can you provide any insight into this?

Reply:
It actually didn't work with rutorrent natively at the time. It could need retesting if rutorrent is updated (there were some command semantic changes which required rutorrent changes last time I checked).